go-controller/pkg/ovn/gateway_init.go

```diff
@@ -537,18 +538,25 @@ func addPolicyBasedRoutes(nodeName, mgmtPortIP string, hostIfAddr *net.IPNet) er
 
 	// embed nodeName as comment so that it is easier to delete these rules later on.
 	// logical router policy doesn't support external_ids to stash metadata
-	matchStr := fmt.Sprintf(`inport == "%s%s" && %s.dst == %s /* %s */`,
-		types.RouterToSwitchPrefix, nodeName, l3Prefix, hostIfAddr.IP.String(), nodeName)
-	if err := syncPolicyBasedRoutes(nodeName, mgmtPortIP, matchStr, types.NodeSubnetPolicyPriority, mgmtPortIP, "match,nexthops"); err != nil {
+	hostIPs := make([]string, len(otherHostAddrs))
```

Comment:
`otherHostAddrs` will include the host's IP right - because that's an IP that ovnkube-node will find an annotate? Which will result in it being added twice, here and on L543?

Reply:
yes and to your comment https://github.com/ovn-org/ovn-kubernetes/pull/2159/files#r610757623 will handle it with sets

I'm just trying to ensure we always add the kube node ip since its something we already know exists.

Reply:
I've gone ahead and changed this.